docs/extend/extend-apiml/api-mediation-onboard-rest-service-direct-eureka-call.md

```diff
@@ -129,21 +129,23 @@ where:
 
 At registration time, provide metadata in the following format. Metadata parameters contained in this code block are described in the following section.
 
-```
+```xml
 <instance>
   <metadata>
       <apiml.catalog.tile.id>samples</apiml.catalog.tile.id>
       <apiml.catalog.tile.title>Sample API Mediation Layer Applications</apiml.catalog.tile.title>
       <apiml.catalog.tile.description>Applications which demonstrate how to make a service integrated to the API Mediation Layer ecosystem</apiml.catalog.tile.description>
       <apiml.catalog.tile.version>1.0.1</apiml.catalog.tile.version>
-      <apiml.service.title>Sample Service ©</apiml.service.title>
```

Comment:
I guess that has been used to test problems with such characters in the API Catalog UI but this just a copy of configuration in the documentation.